src/Functional.CQS.AOP.Caching.Infrastructure.DistributedCache.Redis/FunctionalRedisCacheConfiguration.cs

```diff
@@ -22,24 +23,25 @@ private FunctionalRedisCacheConfiguration(string hostURL, int portNumber, params
 		/// </summary>
 		/// <param name="jsonConverterCollection">The collection of custom JSON converters.</param>
 		public static FunctionalRedisCacheConfiguration ForLocalHostPort6379(params JsonConverter[] jsonConverterCollection)
-			=> new FunctionalRedisCacheConfiguration(LOCALHOST, 6379, jsonConverterCollection);
+			=> new FunctionalRedisCacheConfiguration(LOCALHOST, 6379, "", jsonConverterCollection);
 
 		/// <summary>
 		/// Initializes a new instance of the <see cref="FunctionalRedisCacheConfiguration"/> class.
 		/// </summary>
 		/// <param name="portNumber">The port number.</param>
 		/// <param name="jsonConverterCollection">The collection of custom JSON converters.</param>
 		public static FunctionalRedisCacheConfiguration ForLocalHost(int portNumber, params JsonConverter[] jsonConverterCollection)
-			=> new FunctionalRedisCacheConfiguration(LOCALHOST, portNumber, jsonConverterCollection);
+			=> new FunctionalRedisCacheConfiguration(LOCALHOST, portNumber, "", jsonConverterCollection);
 
 		/// <summary>
 		/// Initializes a new instance of the <see cref="FunctionalRedisCacheConfiguration"/> class.
 		/// </summary>
 		/// <param name="hostURL">The host URL.</param>
 		/// <param name="portNumber">The port number.</param>
+		/// <param name="password">The password for the Redis connection.</param>
 		/// <param name="jsonConverterCollection">The collection of custom JSON converters.</param>
-		public static FunctionalRedisCacheConfiguration ForRemoteHost(string hostURL, int portNumber, params JsonConverter[] jsonConverterCollection)
-			=> new FunctionalRedisCacheConfiguration(hostURL, portNumber, jsonConverterCollection);
+		public static FunctionalRedisCacheConfiguration ForRemoteHost(string hostURL, int portNumber, string password, params JsonConverter[] jsonConverterCollection)
```

Comment:
It is possible that I have not noticed it in the diff, but you will have to keep the old `public static FunctionalRedisCacheConfiguration ForRemoteHost(string hostURL, int portNumber, params JsonConverter[] jsonConverterCollection)` factory method in addition to the new one, otherwise it's a breaking change.

Please increment the minor version of this package.

Reply:
Ah yes, that's right. Thanks Ryan.